x-pack/plugins/enterprise_search/public/applications/workplace_search/views/groups/constants.ts

```diff
@@ -0,0 +1,20 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { i18n } from '@kbn/i18n';
+
+export const NAV = {
```

Comment:
@scottybollinger Feel free to rename this constant later to whatever you prefer - I was going to ask you about that & get your thoughts but since we're in a bit of a rush due to FF / breaking Kibana types, it probably needs to be a separate PR. Apologies!